saleor/plugins/webhook/utils.py

```diff
@@ -222,6 +222,23 @@ def parse_list_shipping_methods_response(
     return shipping_methods
 
 
+def parse_tax_codes(
+    response_tax_data: Any,
+) -> Optional[List[TaxType]]:
+    if not isinstance(response_tax_data, list):
+        return None
+    try:
+        return [
+            TaxType(
+                code=tax["code"],
+                description=tax["description"],
+            )
+            for tax in response_tax_data
+        ]
+    except KeyError:
+        return None
```

Comment:
I will be a badass here but it looks like hieroglyph :D Do we really need that list comprehensions in `try...except` ?  Consider classic old schoold for loop.